dali/test/python/input/test_video.py

```diff
@@ -31,7 +32,15 @@
 
 batch_size_values = [1, 3, 100]
 frames_per_sequence_values = [1, 7, 100]
-device_values = ['cpu']
+device_values = ['cpu', 'mixed']
+
+
+def is_gpu_tensor(tensor):
+    return isinstance(tensor, dali.backend_impl.TensorListGPU)
+
+
+def always_cpu_array(dali_tensor):
+    return dali_tensor.as_cpu().as_array() if is_gpu_tensor(dali_tensor) else dali_tensor.as_array()
```

Comment:
please use `test_utils.as_array()`. It does exactly this, I believe

Reply:
Nice, I didn't know it exists :)